README.md

```diff
@@ -93,6 +93,21 @@ $ grunt prod
 
 * explore `config/env/production.js` for production environment configuration options
 
+### Running with User Seed
+To have an 'admin' and 'user' user account seeded at runtime:
```

Comment:
May be: "To have default account(s) seeded at runtime". Since admin is not created in development mode